.github/actions/configure_wine/action.yml

```diff
@@ -0,0 +1,76 @@
+name: 'Configure Wine'
+description: 'Crossover-version independent action to configure Wine'
+inputs:
+  winearch:
+    description: 'wine64 or wine32on64'
+    required: true
+    default: "wine64"
+  crossflags:
+    descriptions: 'CROSSFLAGS to be given to ming-w64'
+    required: true
+    default: "-g -O2"
+  cflags:
+    descriptions: 'CFLAGS to be given to clang'
+    required: true
+    # Xcode12 by default enables '-Werror,-Wimplicit-function-declaration' (49917738)
+    # this causes wine(64) builds to fail so needs to be disabled.
+    # https://developer.apple.com/documentation/xcode-release-notes/xcode-12-release-notes    
+    default: "-g -O2 -Wno-implicit-function-declaration -Wno-deprecated-declarations -Wno-format"
+  ldflags:
+    descriptions: 'LDFLAGS to be given to clang'
+    required: true
+    default: "-Wl,-headerpad_max_install_names"
+  build-dir:
+    description: 'Build directory'
+    required: true
+  wine-configure:
+    description: 'Path to wine configure script'
+    required: true
+    default: ${{ github.workspace }}/sources/wine/configure
+  configure-params:
+    description: 'additional parameters to be given to the wine configure script'
+    required: true
+    default: "--enable-win64 --with-vulkan"
+runs:
+  using: "composite"
+  steps:
+      - name: Configure ${{ inputs.winearch }}
+        env:
+          CC: clang
+          CXX: clang++
+          CROSSCFLAGS: ${{ inputs.crossflags }}
+          CFLAGS: ${{ inputs.cflags }}
+          LDFLAGS: ${{ inputs.ldflags }}
+        run: |
+          set -x
+
+          export SDL2_CFLAGS="-I$(brew --prefix sdl2)/include -I$(brew --prefix sdl2)/include/SDL2"
```

Comment:
This workaround isn’t needed for CX21, however it could also be skipped entirely by adapting the upstream patch https://github.com/wine-mirror/wine/commit/e4fbae832c868e9fcf5a91c58255fe3f4ea1cb30.patch

Reply:
I prefer not to depend on the actual crossover-version inside the action, so that differences between the invocations are obvious directly inside `build_monolithic.yml`. Thus, it's either about properly applying that patch or just leaving things as they are here (doesn't hurt much, I think...). - You are seeing me undecided :-D

Reply:
It doesn’t make much difference just an alternative.